express/app/controllers/users.js

```diff
@@ -0,0 +1,16 @@
+const user = require('../models/user');
```

Comment:
This doesn't work. You have to do this: `const user = require('../models').users`